packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -1115,7 +1115,10 @@ function detachFiber(current: Fiber) {
   current.pendingProps = null;
   current.memoizedProps = null;
   current.stateNode = null;
+  current.alternate = null;
```

Comment:
This is already set earlier.

Reply:
LOL

Reply:
I removed that line. I'll keep the change Brian mentioned.

Reply:
What about nextEffect? That one isn't cleared. Would it help to clear that one? The leak I'm observing seems that nextEffect is an important link in the (alternate->nextEffect) chain to get to Fiber that are much older than just the previous update.

Could the leak just be that we're missing that one?

Reply:
We can't clear that field here as it's used for traversal to the next effect. If we cleared it, it would break the reconciler work loop. I'm looking into attempting to try and clear it another way now.